nemo/core/classes/modelPT.py

```diff
@@ -39,6 +39,7 @@ class ModelPTConfig(Config):
     train_ds: Optional[DictConfig] = None
     validation_ds: Optional[DictConfig] = None
     test_ds: Optional[DictConfig] = None
+    pl: Optional[DictConfig] = None
```

Comment:
we'll drop pl once https://github.com/NVIDIA/NeMo/pull/848 is merged. Keep for now